src/platformdirs/windows.py

```diff
@@ -103,12 +133,14 @@ def get_win_folder_from_registry(csidl_name: str) -> str:
     registry for this guarantees us the correct answer for all CSIDL_*
     names.
     """
-    shell_folder_name = {
-        "CSIDL_APPDATA": "AppData",
-        "CSIDL_COMMON_APPDATA": "Common AppData",
-        "CSIDL_LOCAL_APPDATA": "Local AppData",
-    }.get(csidl_name)
-    if shell_folder_name is None:
+    try:
+        shell_folder_name = {
+            "CSIDL_APPDATA": "AppData",
+            "CSIDL_COMMON_APPDATA": "Common AppData",
+            "CSIDL_LOCAL_APPDATA": "Local AppData",
+            "CSIDL_PERSONAL": "Personal",
+        }[csidl_name]
+    except KeyError:
         raise ValueError(f"Unknown CSIDL name: {csidl_name}")
```

Comment:
This feels YAGNI.

Reply:
The try except?

Reply:
Yes, I don't follow what we need these changes on your Windows 🤔

Reply:
I just changed it to a try except because it is more efficient for cases that don't happen often compared with using an if every time. It was just a passing change as I worked with the file, but I can revert it. I don't think it's important enough for a new PR.

Reply:
Yes 🙏